splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -324,6 +324,16 @@
           "type": "string",
           "description": "Link to external TA documentation used for redirection on error dashboard page",
           "format": "uri"
+        },
+        "settings": {
+          "type": "object",
+          "description": "Additional settings for monitoring dashboard.",
+          "properties": {
+            "custom_tab_name": {
```

Comment:
Should there be a maximum length that we can support?

Reply:
hmmm we can but at the same time in splunk-ui there isn't any length restriction. Also at the same time if someone wants their addon to have a very long name in a tab i do not see a reason to not let him do that. In worst case (on image) width of rest elements is reduced and ... are added at the end (only beginning part of long title is displayed also).
![Screenshot 2024-06-18 at 16 55 19](https://github.com/splunk/addonfactory-ucc-generator/assets/143183665/4fab3b4e-50aa-44db-91b1-2cada6fe3bb7)